src/sparseml/export/export.py

```diff
@@ -207,16 +196,29 @@ def export(
         source_path=source_path,
         target_path=target_path,
         deployment_directory_name=deployment_directory_name,
-        deployment_directory_files=helper_functions.deployment_directory_structure,
+        deployment_directory_files_mandatory=helper_functions.deployment_directory_files_mandatory,  # noqa: E501
+        deployment_directory_files_optional=helper_functions.deployment_directory_files_optional,  # noqa: E501
         onnx_model_name=onnx_model_name,
     )
+
+    _LOGGER.info(
+        f"Applying optimizations: {graph_optimizations} to the exported model..."
+    )
+    apply_optimizations(
```

Comment:
Since some optimizations (like kv cache injection), need access to the additional files, not only model.onnx, I have decided to first create the deployment directory (copy all the appropriate files), and only then move into applying optimizations to the model.